Maestro/subscriptions.json

```diff
@@ -405,6 +426,27 @@
             "/verbosity:Normal"
           ]
         },
+        // This handler will bring the Latest CoreCLR release/1.1.0 build into CoreCLR release/1.1.0
```

Comment:
Is this about bringing External CoreCLR dependencies (i.e. the ones from TFS build) into CoreCLR 1.1.0 branch?